platform/posix/iot_logging.c

```diff
@@ -28,10 +28,15 @@
 /* Standard includes. */
 #include <stdarg.h>
 #include <stdio.h>
+#include <stdlib.h>
 #include <string.h>
+#include <assert.h>
 
 /* Platform clock include. */
-#include "platform/iot_clock.h"
+#include "platform/include/iot_clock.h"
+
+/* Include header for logging level macros. */
+#include "iot_logging_levels.h"
 
 /* Logging includes. */
 #include "iot_logging.h"
```

Comment:
platform/include/iot_logging.h?
I mean they are in the same place?

Reply:
Updated